test/integration/pj-rehearse/expected.yaml

```diff
@@ -1059,262 +1059,6 @@
-      grace_period: 15s
-      skip_cloning: true
-      timeout: 4h0m0s
-      utility_images:
-        clonerefs: gcr.io/k8s-prow/clonerefs:v20190129-0a3c54c
-        entrypoint: gcr.io/k8s-prow/entrypoint:v20190129-0a3c54c
-        initupload: gcr.io/k8s-prow/initupload:v20190129-0a3c54c
-        sidecar: gcr.io/k8s-prow/sidecar:v20190129-0a3c54c
-    extra_refs:
-    - base_ref: master
-      org: super
-      repo: duper
-      workdir: true
-    job: rehearse-1234-pull-ci-super-duper-master-test-template-e2e-openstack
-    namespace: test-namespace
-    pod_spec:
-      containers:
-      - args:
-        - --artifact-dir=$(ARTIFACTS)
-        - --give-pr-author-access-to-namespace=true
-        - --secret-dir=/usr/local/e2e-aws-cluster-profile
-        - --target=e2e-aws
-        - --template=/usr/local/e2e-aws-openstack
-        command:
-        - ci-operator
-        env:
-        - name: CLUSTER_TYPE
-          value: openstack
-        - name: JOB_NAME_SAFE
-          value: e2e-aws
-        - name: TEST_COMMAND
-          value: TEST_SUITE=openshift/conformance/parallel run-tests
-        - name: CONFIG_SPEC
-          value: |
-            base_images:
-              base:
-                name: origin-v4.0
-                namespace: openshift
-                tag: base
-            build_root:
-              image_stream_tag:
-                name: release
-                namespace: openshift
-                tag: golang-1.10
-            images:
-            - from: base
-              to: test-image
-            resources:
-              '*':
-                limits:
-                  cpu: 500Mi
-                requests:
-                  cpu: 10Mi
-            tag_specification:
-              name: origin-v4.0
-              namespace: openshift
-            zz_generated_metadata:
-              branch: master
-              org: super
-              repo: duper
-        image: ci-operator:latest
-        imagePullPolicy: Always
-        name: ""
-        resources:
-          limits:
-            cpu: 500m
-          requests:
-            cpu: 10m
-        volumeMounts:
-        - mountPath: /usr/local/e2e-openstack-cluster-profile
-          name: cluster-profile
-        - mountPath: /usr/local/e2e-openstack
-          name: job-definition
-          subPath: test-template.yaml
-      serviceAccountName: ci-operator
-      volumes:
-      - name: cluster-profile
-        projected:
-          sources:
-          - secret:
-              name: cluster-secrets-openstack
-      - configMap:
-          name: rehearse-template-test-template-4b355dc3
-        name: job-definition
-    refs:
-      base_ref: master
-      base_sha: test_sha
-      org: openshift
-      pulls:
-      - author: petr-muller
-        number: 1234
-        sha: test_sha
-      repo: release
-    report: true
-    rerun_command: /test pj-rehearse
-    type: presubmit
-  status:
-    startTime: 2020-06-22T22:25:00Z
-    state: triggered
```

Comment:
These two jobs are the same case: they are not actually using any of the CMs that will be updated by this PR, so new rehearse sees no reason to even rehearse them (see their configs: [1](https://github.com/openshift/ci-tools/blob/921be048c697211820d3c5afbaec88d8e8273fe9/test/integration/pj-rehearse/master/ci-operator/jobs/super/duper/super-duper-master-presubmits.yaml#L196-L247) [2](https://github.com/openshift/ci-tools/blob/921be048c697211820d3c5afbaec88d8e8273fe9/test/integration/pj-rehearse/master/ci-operator/jobs/super/duper/super-duper-master-presubmits.yaml#L144-L195))